net/redisclient.go

```diff
@@ -226,11 +248,67 @@ func NewRedisRingClient(ro *RedisOptions) *RedisRingClient {
 		r.ring = redis.NewRing(ringOptions)
 		r.log = ro.Log
 		r.metricsPrefix = ro.MetricsPrefix
+
+		if ro.AddrUpdater != nil {
+			if ro.UpdateInterval == 0 {
+				ro.UpdateInterval = defaultUpdateInterval
+			}
```

Comment:
minor: maybe move this into `startUpdater`

Reply:
I don't like to put it there because IMO options/defaults should be set in one place.